### PR TITLE
Svelte & web merge

### PR DESCRIPTION
**Pull request by Denperidge**
Generally notable changes are in 
app/svelte/svelte.config.js
app/svelte/tsconfig.json
app/svelte/vite.config.js

### CODE REVIEW

File: app/svelte/src/routes/index.svelte [hunk 1]
```diff
@@ -2,12 +2,112 @@
...
+		console.log('get data');
+		getData();
+	}
+
+	/**
+	 * Get data from pod
+	 */
+	async function getData() {
+		// https://docs.inrupt.com/developer-tools/javascript/client-libraries/tutorial/read-write-data/#write-a-new-soliddataset
+		let dataset: any;
```
Comment (yustarandomname) [inline app/svelte/src/routes/index.svelte:79]: This `any` type is a bit sketchy, however I could not change it to anything more specific, or else some other could would start complaining

File: app/svelte/svelte.config.js [hunk 2]
```diff
@@ -1,14 +1,32 @@
-import adapter from '@sveltejs/adapter-auto';
+import adapter from '@sveltejs/adapter-static';
 import preprocess from 'svelte-preprocess';
 
+const dev = process.env.NODE_ENV === 'development';
```
[the remark this answers is not in the document]
Comment (yustarandomname) [reply app/svelte/svelte.config.js:4]: Aparently this will not  work

File: app/svelte/src/routes/index.svelte (hunk 1 above)
Comment (Denperidge) [reply app/svelte/src/routes/index.svelte:79]: Oh you're 100% right. I tried to use : function but that's not allowed apparently 😭

File: app/svelte/svelte.config.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Denperidge) [reply app/svelte/svelte.config.js:4]: Oh yeah no, its not about npm run dev, but about setting the NODE_ENV environment variable. At least that's the case in the original code, Any idea how it works with the dev built into svelte?

**Review by yustarandomname — APPROVED** [review]

**Pull request merged**